backend/build.gradle

```diff
@@ -79,6 +79,24 @@ dependencies {
 
     implementation 'com.azure:azure-storage-queue:12.12.+'
 
+    implementation 'io.netty:netty-codec:4.1.73.Final'
```

Comment:
is this managed internally by Spring now? or are these new dependencies?

Reply:
Spring manages these.  All of this is unnecessary when https://github.com/CDCgov/prime-simplereport/pull/3562 is complete.  Until then, forcing these versions fixes 11 vulnerabilities.